### PR TITLE
app-doc/cppman: Bump to 0.5.4

### PR DESCRIPTION
**Pull request by mid-kid**
- app-doc/cppman: Pick up maintainance
- app-doc/cppman: Bump to 0.5.4
- app-doc/cppman: Drop old

### CODE REVIEW

File: app-doc/cppman/cppman-0.5.4.ebuild [hunk 1]
```diff
@@ -1,12 +1,12 @@
-# Copyright 1999-2021 Gentoo Authors
+# Copyright 1999-2022 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=8
 
-PYTHON_COMPAT=( python3_{8,9,10} )
+PYTHON_COMPAT=( python3_{8,9,10,11} )
```
Comment (thesamesam) [inline app-doc/cppman/cppman-0.5.4.ebuild:6]:
```suggestion
PYTHON_COMPAT=( python3_{8..11} )
```

Comment (mid-kid) [reply app-doc/cppman/cppman-0.5.4.ebuild:6]: good catch

**Comment by thesamesam** [comment]
Thanks - could you rebase then we'll get this in? cc @Flowdalic as did bump too, in case interested/can give better review

**Comment by mid-kid** [comment]
I've bumped it, but if @Flowdalic is interested in maintaining this too, I'll add them to the maintainer file.

**Review by Flowdalic — APPROVED** [review]
utACK. Thanks for the fixes. 👍 
I am not interested at this moment in maintaining cppman. I do consider migrating stdman from ::guru to ::gentoo, as I am more fond of stdman installing the man pages system-wide, which, I believe, cppman does not support.

**Comment by mid-kid** [comment]
Seconding the notion of moving stdman, it's honestly a much, much better approach than what cppman is doing, as I much prefer having local pages. However, cppreference.com really wasn't structured to be a man page, and `cppman`'s vim pager can follow links with `K`, making it a much nicer browsing experience, so I think I'll be using `cppman` for the time being.

File: app-doc/cppman/cppman-0.5.4.ebuild [hunk 2]
```diff
@@ -29,5 +29,8 @@ src_prepare() {
 	default
 
 	# Don't allow setup.py to install documentation directly
-	sed -i '\:share/doc/cppman:d' setup.py || die "sed failed"
+	sed -i '\:share/doc/cppman:d' setup.py || die
+
+	# Fix zsh completion directory
+	sed -i 's:share/zsh/vendor-completions/:share/zsh/site-functions/:' setup.py || die
```
Comment (Flowdalic) [inline app-doc/cppman/cppman-0.5.4.ebuild:35]: I believe this requires a revision bump

**Review by Flowdalic — CHANGES_REQUESTED** [review]
FWIW, I just noticed that the ebuilds emits the following QA warning:

```
 * Problems with installed bash completions were found:
 * 
 * 	cppman.bash: incorrect name, no completions for 'cppman.bash' command defined.
 * 	cppman: missing alias (symlink) for completed command.
```

While those already exists in the existing ebuild, would you be so kind and try to fix those too?

File: app-doc/cppman/cppman-0.5.4.ebuild (hunk 2 above)
Comment (mid-kid) [reply app-doc/cppman/cppman-0.5.4.ebuild:35]: You're correct, I didn't forsee the version bump so forgot to do it :+1:

I'll fix both of these things tonight, might take a bit to get home.

**Pull request closed without merging**